src/job-execution/WebJobExecutor.ts

```diff
@@ -169,13 +169,18 @@ export class WebJobExecutor extends BaseJobExecutor {
               ? res.result.log.map((logLine: any) => logLine.line).join('\n')
               : res.result.log
 
-          const resObj =
-            this.serverType === ServerType.Sasjs
-              ? {
-                  result: res.result._webout,
-                  log: parsedSasjsServerLog
-                }
-              : res
+          let resObj = res
+
+          if (this.serverType === ServerType.Sasjs) {
+            resObj = {
+              result: res.result._webout,
+              log: parsedSasjsServerLog
+            }
+
+            if (resObj.result.length < 1)
+              throw new JobExecutionError(0, 'Job execution failed', resObj)
```

Comment:
So, sasjs/server will always return some webout?

Reply:
In resObj even if no webout, there will be a log property so we take it if exists to append to sasjs requests

Reply:
@allanbowe This bit is part of the fix where requests modal is empty. Log not beign added.

Reply:
comment by Allan:
> If it’s a Job, there will never be a webout.  If it’s a Service or a Test,
there will always be a webout.